circle.yml

```diff
@@ -2,7 +2,7 @@ version: 2
 jobs:
   build:
     docker:
-      - image: circleci/node:8.11-browsers
+      - image: circleci/node:10.13.0-browsers
```

Comment:
I will change this back. We had a discussion about this and it should be 8.11 for now.